### PR TITLE
[8 of 8: Service Updates & Upgrades] Combine all ServicePlan related components into working UI

### PR DESCRIPTION
**Pull request by jfurrow**
This is the big one that relies on all of the following:
- [x] #195 (2 of 8)
- [x] #196 (3 of 8) 
- [x] #197 (4 of 8)
- [x] #198 (5 of 8)
- [x] #199 (6 of 8)
- [x] #200 (7 of 8)

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by daiyiwhale** [comment]
Oh no you already got some conflicts ): ):

**Comment by jfurrow** [comment]
I'm adding `hold merge` to this because I need to do some refactoring in order to properly use the new `PageHeader` component.

**Comment by jfurrow** [comment]
I just rebased `feature-branch/software-updates` off master, and then rebased this branch off `feature-branch/software-updates`, so everything is up to date! (for now...)

File: src/js/components/ServiceDetail.js [hunk 1]
```diff
@@ -23,8 +31,49 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
...
+    }];
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  componentDidMount() {
+    super.componentDidMount(...arguments);
+    ServicePlanStore.addPlanChangeListener(this.props.service.id,
```
Comment (kennyt) [inline src/js/components/ServiceDetail.js]: we don't usually indent it like this. one variable per line

File: src/js/components/ServiceDetail.js [hunk 2]
```diff
@@ -23,8 +31,49 @@ class ServiceDetail extends mixin(InternalStorageMixin, TabsMixin) {
...
+      this.onServicePlanStoreChange);
+  }
+
+  componentWillUnmount() {
+    super.componentWillUnmount(...arguments);
+    ServicePlanStore.removePlanChangeListener(this.props.service.id,
```
Comment (kennyt) [inline src/js/components/ServiceDetail.js]: we don't usually indent it like this. one variable per line

File: src/js/components/ServiceDetail.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/ServiceDetail.js]: Done!

File: src/js/components/ServiceDetail.js (hunk 2 above)
Comment (jfurrow) [inline src/js/components/ServiceDetail.js]: Done!

**Comment by jfurrow** [comment]
retest this please

**Pull request merged**